Cargo.toml

```diff
@@ -88,6 +90,7 @@ features = [
   "WheelEvent",
   "Window",
   "Worker",
+  "WorkerGlobalScope",
```

Comment:
nit: remove

Reply:
I have to move this to #867 then.
I thought not touching `Cargo.toml` in the other PR's made sense so I don't have to rebase all the time.

Reply:
Yeah let's move it to #867. Relevant changes should be made together 👍

Reply:
Sure.